buttercup.el

```diff
@@ -267,7 +279,7 @@ MATCHER is either a matcher defined with
   failure-stack)
 
 (defun buttercup-suite-add-child (parent child)
-  "Add a CHILD suite to a PARENT suite."
+  "Add to PARENT suite a CHILD suite."
```

Comment:
I know the Emacs linter requires args in docstrings to be of the same order as in the definition, but this is just extremely awkward. I don't think this is a good docstring.